src/js/hca-rjsf/config/form.js

```diff
@@ -147,22 +163,146 @@ const formConfig = {
           path: 'veteran-information/demographic-information',
           title: 'Veteran information',
           initialData: {},
-          uiSchema: {},
-          schema: {}
+          uiSchema: {
+            gender: {
+              'ui:title': 'Gender'
+            },
+            maritalStatus: {
+              'ui:title': 'Marital status'
+            },
+            'view:demographicCategories': {
+              'ui:title': 'Which categories best describe you?',
+              'ui:description': 'You may check more than one.',
+              isSpanishHispanicLatino: {
+                'ui:title': 'Spanish, Hispanic, or Latino'
+              },
+              isAmericanIndianOrAlaskanNative: {
+                'ui:title': 'American Indian or Alaskan Native'
+              },
+              isBlackOrAfricanAmerican: {
+                'ui:title': 'Black or African American'
+              },
+              isNativeHawaiianOrOtherPacificIslander: {
+                'ui:title': 'Native Hawaiian or Other Pacific Islander'
+              },
+              isAsian: {
+                'ui:title': 'Asian'
+              },
+              isWhite: {
+                'ui:title': 'White'
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            required: ['gender', 'maritalStatus'],
+            properties: {
+              gender: {
+                type: 'string',
+                'enum': genders.map(gender => gender.value),
+                enumNames: genders.map(gender => gender.label)
+              },
+              maritalStatus: {
+                type: 'string',
+                'enum': maritalStatuses
+              },
+              'view:demographicCategories': {
+                type: 'object',
+                properties: {
+                  isSpanishHispanicLatino,
+                  isAmericanIndianOrAlaskanNative,
+                  isBlackOrAfricanAmerican,
+                  isNativeHawaiianOrOtherPacificIslander,
+                  isAsian,
+                  isWhite
+                }
+              }
+            }
+          }
         },
         veteranAddress: {
           path: 'veteran-information/veteran-address',
           title: 'Permanent address',
           initialData: {},
-          uiSchema: {},
-          schema: {}
+          uiSchema: {
+            address: _.merge(address.uiSchema('Permanent address'), {
+              street3: {
+                'ui:title': 'Line 3'
+              }
+            })
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              address: _.merge(address.schema(true), {
+                properties: {
+                  street: {
+                    type: 'string',
+                    minLength: 1,
+                    maxLength: 30
```

Comment:
Current HCA has these maxLenghths for certain address fields due to ES requirements.